appservice/package-lock.json

```diff
@@ -26,7 +26,7 @@
                 "portfinder": "^1.0.25",
                 "pretty-bytes": "^5.3.0",
                 "simple-git": "1.132.0",
-                "vscode-azureextensionui": "^0.46.0",
+                "vscode-azureextensionui": "file:../ui/vscode-azureextensionui-0.47.0.tgz",
```

Comment:
Is this supposed to be set to local?

Reply:
I will update it as soon as 0.47.0 is released. That is why I said "Depends on #963" in the PR description. Also the PR build will fail until I fix it, so there's no risk of merging it on accident.